server/lib/validation.js

```diff
@@ -29,6 +30,7 @@ const TYPES = {
   BOOLEAN: joi.boolean(),
   DIMENSION: joi.number().integer().min(0),
   DOMAIN: joi.string().max(32).regex(PATTERNS.DOMAIN),
+  EXPERIMENT: joi.any().valid(EXPERIMENT_NAMES),
```

Comment:
You can change the `any` to a `string` here. Doesn't make much different in practice because we're limiting values with `valid` anyway, but it's a more useful hint to people reading the code if nothing else.